modules/tinymce/tools/tasks/emojis.js

```diff
@@ -0,0 +1,48 @@
+const resolve = require('resolve');
+const twemoji = require('twemoji');
+
+const prettyPrint = (obj) => JSON.stringify(obj,null, 2);
+
+const generateContent = (json, attribution) => {
+  const header = 'window.tinymce.Resource.add(\'tinymce.plugins.emoticons\', ';
```

Comment:
Feel free to ignore, but would this be cleaner as a template string

Reply:
This isn't transpiled code, so we're limited in what we can use.

Reply:
I was imagining just `window.tinymce.Resource.add('tinymce.plugins.emoticons', ${json})` — not the whole thing. Also while it isn't transpiled, it's run on the latest node (afaik) so things like template strings should be supported. Not super fussed on this change either way though

Reply:
It's not run on the latest node version, it's generally the LTS (though has to work on Node 10+). Having said that, all of those I'm pretty sure support template strings. I do like the idea of just pulling in the JSON though, so will change it to that.

- [ ] Change

Reply:
Fixed